### ISSUE TITLE
Topic - Breast Size

### ISSUE THREAD

**Issue by TheOneandOnlyDargonite** [NickWildish/Mas-NSFW-Submod#58]
Similar to the Penis Size topic While it is inevitable that the majority of those using this submod will be male, I think it would be good to have an equivalent for those that have breasts.

**Comment by mizuotanu-nirera** [comment]
It can be done - the size topic's logic can easily be repurposed to input something like a bust/hip/waist measurement. Alternatively, there could be a choice menu for cup (letter) size. It would also be cool if we came up with an approximation for Monika's own size, so she could comment on how she measures up compared to the player.

**Comment by NickWildish** [comment]
I was speaking to a female friend of mine about this recently, and she was saying that along with cup size they have a number that reperesents their shirt size, so something like D-cup size 10. Do you think we need to get that specific or can we get away with only the cup size?

**Comment by TheOneandOnlyDargonite** [comment]
In comparison to the 'length' topic, the easily accessible categories make this so we don't need to be as specific in my mind.

**Comment by mizuotanu-nirera** [comment]
Indeed. It could be something as simple as "are yours smaller, the same, or bigger than mine?" so she can get a basic idea. If we did want to do something with cup size, I think it would be safe to omit the numeric value for band size - but we would have to be careful as apparently the US, UK, and Australia have actually different sizes for any given letter.
